content/applications/general/payment_acquirers/stripe.rst

```diff
@@ -14,13 +14,31 @@ Configuration
 Credentials tab
 ---------------
 
-Odoo needs your **API Credentials** to connect with your Stripe account, which comprises:
+On premise or on Odoo SH, Odoo needs your **API Credentials** to connect with your Stripe account,
+which comprises:
 
 - :ref:`Publishable Key <stripe/api_keys>`: The key solely used to identify the account with Stripe.
 - :ref:`Secret Key <stripe/api_keys>`: The key to sign the merchant account with Stripe.
 - :ref:`Webhook Signing Secret <stripe/webhook>`: When you enable your webhook on your Stripe
   account, this signing secret must be set to authenticate the messages sent from Stripe to Odoo.
 
+You'll be able to easily create your account by using the **Connect Stripe** button. Once you come
+back on your database from the onboarding :
+
+1) Copy/paste your Publishable and Secret keys (see :ref:`below <stripe/api_keys>`);
+2) Generate your webhook by clicking the **Generate your webhook** button.
```

Comment:
Try not to use numbers when doing a numbered list but use this #. instead, the numbers are then automatically added. It helps if there are any changes [https://www.odoo.com/documentation/15.0/contributing/documentation/rst_cheat_sheet.html#numbered-list](url)
```suggestion
#. Copy/paste your *Publishable* and *Secret* keys (see :ref:`below <stripe/api_keys>`).
#. Generate your webhook by clicking on the **Generate your webhook** button.
```